src/Bridge/Symfony/DependencyInjection/Compiler/AdapterCompilerPass.php

```diff
@@ -36,10 +36,12 @@ public function process(ContainerBuilder $container): void
             $container->removeDefinition('sonata.doctrine.adapter.doctrine_orm');
         }
 
-        if ($container->has('doctrine_phpcr')) {
-            $definition->addMethodCall('addAdapter', [new Reference('sonata.doctrine.adapter.doctrine_phpcr')]);
-        } else {
-            $container->removeDefinition('sonata.doctrine.adapter.doctrine_phpcr');
+        if ($container->has('sonata.doctrine.adapter.doctrine_phpcr')) {
```

Comment:
IMO we should have the same code we use for the ORM ob the above lines

Reply:
$container->has('doctrine_phpcr') && $container->has('sonata.doctrine.adapter. doctrine_phpcr');

Reply:
After current change in `SonataDoctrineExtension` this will be enought:
```php
if ($container->has('sonata.doctrine.adapter.doctrine_phpcr')) {
    $definition->addMethodCall('addAdapter', [new Reference('sonata.doctrine.adapter.doctrine_phpcr')]);
}
```